documentation/src/main/wiki/BacklogAndReleasing.md

```diff
@@ -0,0 +1,44 @@
+# Backlog and Releasing
+
+## Release Semantic Versioning
+Knot.x releases follow [Semantic Versioning 2.0.0](https://semver.org/) guide.
+Each release has a version number `MAJOR.MINOR.PATCH`. Those numbers are incremented when:
+- **MAJOR** version when Knot.x introduce incompatible API changes or major architecture refactoring,
+- **MINOR** version when Knot.x introduces new complex functionality in a backwards-compatible manner, and
+- **PATCH** version when Knot.x introduces backwards-compatible bug fixes, small improvements or required dependency update.
+
+## When to migrate
+- **MAJOR** - new project start or planned migration - migration may require a significant effort.
+- **MINOR** - if you need some improvements that are in this release, planned migration - migration 
+may require a minor effort.
+- **PATCH** - as often as possible, no compatibility break, only bugfixes or very small improvements 
+that does not change any system logic - migration should not take any effort.
+
+
+## Bugfixes support and releasing
+Knot.x as an Open Source project supports **the last MINOR** release with bugfixes released regularly 
```

Comment:
Please add that we recommend that users should regularly update to the latest minor version.